esphome/components/nextion/nextion.cpp

```diff
@@ -162,21 +162,24 @@ void Nextion::dump_config() {
                 "  Wake On Touch:    %s\n"
                 "  Exit reparse:     %s",
                 YESNO(this->auto_wake_on_touch_), YESNO(this->exit_reparse_on_start_));
+#ifdef USE_NEXTION_MAX_COMMANDS_PER_LOOP
+  ESP_LOGCONFIG(TAG, "  Max commands per loop: %u", this->max_commands_per_loop_);
+#endif  // USE_NEXTION_MAX_COMMANDS_PER_LOOP
 
   if (this->touch_sleep_timeout_ != 0) {
-    ESP_LOGCONFIG(TAG, "  Touch Timeout:    %" PRIu32, this->touch_sleep_timeout_);
+    ESP_LOGCONFIG(TAG, "  Touch Timeout:         %" PRIu32, this->touch_sleep_timeout_);
   }
 
   if (this->wake_up_page_ != -1) {
-    ESP_LOGCONFIG(TAG, "  Wake Up Page:     %" PRId16, this->wake_up_page_);
+    ESP_LOGCONFIG(TAG, "  Wake Up Page:          %" PRId16, this->wake_up_page_);
```

Comment:
There's no need to use the macros for sizes other than 32 (we're not doing so anywhere else).
```suggestion
    ESP_LOGCONFIG(TAG, "  Wake Up Page:          %d", this->wake_up_page_);
```